lib/colord/cd-test-shared.c

```diff
@@ -42,6 +42,7 @@ cd_test_get_filename (const gchar *filename)
 	gchar *tmp;
 	char full_tmp[PATH_MAX];
 	g_autofree gchar *path = NULL;
+	const gchar *testdatadir = NULL;
```

Comment:
it doesn't need the `=  NULL` and needs to be moved above the g_autofree please too.